src/client/components/search/search.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ToggleGroup from '../utils/ToggleGroup';
+
+function fetchPosts() {
+    fetch('https://jsonplaceholder.typicode.com/posts')
+        .then((response) => {
+            return response.json();
+        })
+        .then((json) => {
+            console.log('json', json);
+        });
+}
+
+function fetchComments() {
+    fetch('https://jsonplaceholder.typicode.com/comments')
+        .then((response) => {
+            return response.json();
+        })
+        .then((json) => {
+            console.log('json', json);
+        });
+}
+
+function fetchPosts() {
+    fetch('https://jsonplaceholder.typicode.com/posts')
+        .then((response) => {
+            return response.json();
+        })
+        .then((json) => {
+            console.log('json', json);
+        });
+}
+
+function fetchComments() {
+    fetch('https://jsonplaceholder.typicode.com/comments')
```

Comment:
@vitkuz those fetches are not used, right?

Reply:
just an example, placeholders for next task